native/cocos/audio/apple/AudioCache.h

```diff
@@ -69,7 +70,9 @@ class AudioCache {
     float _duration;
     uint32_t _totalFrames;
     uint32_t _framesRead;
-
+    uint32_t _bytesPerFrame;
```

Comment:
Initialize member variables.

Reply:
These variables are initd in the constructor, and default value is 0.